File: 240327/softeer-6269.py
M, N, K = map(int, input().split())

secret_number = [int(num) for num in input().split()]
user_number = [int(num) for num in input().split()]

flag = False
for i in range(0,N-M+1):
    cnt = 0
    for j in range(0,M):
        if secret_number[j] == user_number[i+j]:
            cnt += 1
        else:
            break
    if cnt == M:
        flag = True

if flag == True:
    print("secret")
else:
    print("normal")

File: 240327/softeer-6269.cpp
#include<iostream>
#include<vector>

using namespace std;

int main(int argc, char** argv)
{
    int M, N, K;
    cin >> M >> N >> K;

    vector<int> secret_number(M);
    for(int i=0; i<M; i++){
        cin >> secret_number[i];
    }

    vector<int> user_number(M);
    for(int i=0; i<N; i++){
        cin >> user_number[i];
    }

    bool flag = false;
    for(int i=0; i<N-M+1; i++){
        int cnt = 0;
        for(int j=0; j<M; j++){
            if(secret_number[j] == user_number[i+j]){
                cnt += 1;
            }else{
                break;
            }
        }
        if(cnt == M){
            flag = true;
        }
    }

    if(flag){
        cout << "secret";
    }else{
        cout << "normal";
    }
    return 0;
}